### ISSUE TITLE
Can you set schema catalogs in config files?

### ISSUE THREAD

**Issue by SamB** [tamasfe/taplo#458]
I couldn't find anything about this in the docs.

(Of course, I the doc structure is kind of hard to follow, especially on mobile where you don't get the top bar navigation...)

**Comment by ia0** [comment]
I don't think you can use another schema catalog than https://www.schemastore.org/api/json/catalog.json at the moment. Do you have a use-case for this?

**Comment by ia0** [comment]
This would have been a possible workaround, but you can also overwrite the schema for just one file as described [here](https://taplo.tamasfe.dev/configuration/using-schemas.html) in particular the [schema directive](https://taplo.tamasfe.dev/configuration/directives.html#the-schema-directive) at the top of the document.
